manifest.toml

```diff
@@ -0,0 +1,103 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "weblate"
+name = "Weblate"
+description.en = "Translation platform using Git and Python"
+description.fr = "Plateforme de traduction utilisant Git et Python"
+
+version = "5.3.1~ynh1"
+
+maintainers = ["Thovi98"]
```

Comment:
@kay0u I proposed to add myself as maintainer as the package was notworking anymore seems few weeks/months and nobody seemed to care about it.
Feel free to make yourself as maintainer :)

Reply:
I maintained this app for a while, but if you want to give some love to this package, feel free to do it!

Reply:
Sure!

Reply:
Done